lib/measured/measurable.rb

```diff
@@ -1,6 +1,8 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
+  PARSE_REGEX = /\A\s*([0-9.]+)\s*([a-zA-Z_-]+)\s*\Z/
```

Comment:
Also in doing that I see that we accept multiple dots. Does that make sense for measured? Numbers/quantities can't have multiple dots.

```ruby
irb(main):001:0> pattern = /
irb(main):002:0/   \A                # start of line
irb(main):003:0/   \s*               # any number of blank characters
irb(main):004:0/   ([0-9.]+)         # one or more digits or decimals
irb(main):005:0/   \s*               # any number of blank characters
irb(main):006:0/   ([a-zA-Z_-]+)     # one or more letters, dashes, or underscores
irb(main):007:0/   \s*               # any number of blank characters
irb(main):008:0/   \Z                # end of line
irb(main):009:0/ /x
=> /
  \A                # start of line
  \s*               # any number of blank characters
  ([0-9.]+)         # one or more digits or decimals
  \s*               # any number of blank characters
  ([a-zA-Z_-]+)     # one or more letters, dashes, or underscores
  \s*               # any number of blank characters
  \Z                # end of line
/x
irb(main):010:0> pattern =~ "1.21.32 gigawatts"
=> 0
```